contracts/GeneralizedTCR.sol

```diff
@@ -403,20 +406,48 @@ contract GeneralizedTCR is IArbitrable, IEvidence {
         if (!round.hasPaid[uint(Party.Requester)] || !round.hasPaid[uint(Party.Challenger)]) {
             // Reimburse if not enough fees were raised to appeal the ruling.
             reward = round.contributions[_beneficiary][uint(Party.Requester)] + round.contributions[_beneficiary][uint(Party.Challenger)];
+        } else if (
+            request.ruling == Party.None &&
+            _round == 0
+        ) {
+            if (
+                _beneficiary != request.parties[uint(Party.Requester)] &&
+                _beneficiary != request.parties[uint(Party.Challenger)]
+            ) return;
+
+            uint side = _beneficiary == request.parties[uint(Party.Requester)] ? 1 : 2;
```

Comment:
We should avoid magic numbers. We can use Party.X and cast it into an uint only at the last moment.